lib/normalizer.js

```diff
@@ -33,7 +33,8 @@ class Normalizer {
 	onField(field, value, subschema, subschemaType) {
 		if (
 			(value === undefined || value === null) &&
-			subschema.default !== undefined && subschema.default !== null
+			subschema.default !== undefined && subschema.default !== null &&
+			!this.options.ignoreDefaults
```

Comment:
Since you're not checking to make sure this.options is non-null here, in the constructor, you should default this.options to empty object.